api/sft/handler.go

```diff
@@ -39,3 +47,57 @@ func (h *Handler) GetAllToggles(w http.ResponseWriter, r *http.Request) {
 
 	utils.WriteJSON(w, toggles)
 }
+
+func (h *Handler) CreateToggle(w http.ResponseWriter, r *http.Request) {
+	fmt.Println("creating toggle")
```

Comment:
Would be good to use log package here - @elliothllm wants to abstract it to its own repo, and we probably should fmt.Println will output to stderr all the time - it's not as controllable as having log.Logger().Trace or .Debug for example where we can exclude it easily from server logs.

Reply:
Cool, have changed to log.Println for now.